packages/cli/src/commands/clean/clean.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import fs from 'fs-extra';
+import { getDefaultCacheOptions } from 'commands/build-cache/options';
+
+const cacheOptions = getDefaultCacheOptions();
+
+export default async function clean() {
+  await fs.remove(cacheOptions.output);
+  await fs.remove(cacheOptions.cacheDir);
```

Comment:
Assuming default config, this will delete `node_modules/.cache/backstage-builds`, but I think it would be better to limit it to only delete the cache for the package that we're cleaning up, which for the cli would be `node_modules/.cache/backstage-builds/packages/cli`. You can see how we figure out the location of the package cache [here](https://github.com/spotify/backstage/blob/3e1e356320c0792cc28f7745aa56ab85c1712f23/packages/cli/src/commands/build-cache/cache.ts#L64)

Reason I'd like it to be a bit more precise is if you're using the clean command in a specific package during development, that's likely because you only want to reset that package, so it would speed up iteration. And the top-level clean would still delete the cache for all packages so it'll behave the same.